scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
 
 export const contractNames = () => ["InvestorVesting"];
 
-const tokenAddress = contracts[process.env.HARDHAT_NETWORK?.toLowerCase()].REMI;
-const startDate = 1660274000;
+const tokenAddress = contracts[process.env.HARDHAT_NETWORK].REMI;
+// Ensure this is a moment in the future!
+
+const startDate = 1662127200;
 
 const vestingSeed = [
   {
-    beneficiary: "0x042BC2D085c0584Bd56D62C170C4679e1ee9FC45",
-    vestingAmount: ethers.BigNumber.from("11150000"),
-    duration: 9000,
+    beneficiary: "0x470D968F0d27075F3Db0f28d2C2a5a6EEbaD9E65",
+    vestingAmount: ethers.BigNumber.from("750000000"),
+    duration: 63072000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.BigNumber.from("300000000"),
```

Comment:
These should all be big numbers right?
```suggestion
    initialAmount: ethers.BigNumber.from("300000000000000000000000000"),
```